docs/en/stores/basic-usage.md

```diff
@@ -0,0 +1,471 @@
+
+const store = new Store<State>();
+const { path, apply } = store;
+const user = { id: '0', name: 'Paul' };
+
+apply([test(path('users', 'current'), undefined), add(path('users', 'current'), user)]);
+```
+
+These operations set the current user unless a current user already exists.
+
+### Commands
+
+Commands are simply functions that are called while executing a process. They return an array of operations that modify the store. Each command is passed a `CommandRequest` which provides `path` and `at` functions to generate `Path`s in a type-safe way, a `get` function for access to the store's state, a `payload` object for the argument that the process executor was called with.
+
+#### The command factory
+
+Stores have a simple wrapper function that acts as a type-safe factory for creating new commands.
+
+> Creating a store factory
+
+```ts
+import { createCommandFactory } from '@dojo/framework/stores/process';
+import { State } from './interfaces';
+
+const createCommand = createCommandFactory<State>();
+
+const myCommand = createCommand(({ at, get, path, payload, state }) => {
+	return [];
+});
+```
+
+`createCommand` will ensure that the wrapped command has the correct typing and the passed `CommandRequest` functions will be typed to the `State` interface provided to `createCommandFactory`. While it is possible to manually type commands this guide will use `createCommand` in its examples.
+
+#### `get`
+
+The `get` function returns a value from the store at a specified path or `undefined` if a value does not exist at that location.
+
+> Using `get`
+
+```ts
+import { createCommandFactory } from '@dojo/framework/stores/process';
+import { State } from './interfaces';
+import { remove, replace } from '@dojo/framework/stores/state/operations';
+
+const createCommand = createCommandFactory<State>();
+
+const updateCurrentUser = createCommand(async ({ at, get, path }) => {
+	const token = get(path('auth', 'token'));
+
+	if (!token) {
+		return [remove(path('users', 'current'))];
+	} else {
+		const user = await fetchCurrentUser(token);
+		return [replace(path('users', 'current'), user)];
+	}
+});
+```
+
+This example checks for the presence of an authentication token and works to update the current user information.
+
+#### `payload`
+
+The `payload` is an object literal passed into a command when it is called from a process. The `payload`'s type may be defined when constructing the command.
+
+> Defining a `payload`
+
+```ts
+import { createCommandFactory } from '@dojo/framework/stores/process';
+import { State } from './interfaces';
+import { remove, replace } from '@dojo/framework/stores/state/operations';
+
+const createCommand = createCommandFactory<State>();
+
+const addUser = createCommand<User>(({ at, path, payload }) => {
+	return [add(at(path('users', 'list'), 0), payload)];
+});
+```
+
+This example adds a user provided in the `payload` to the beginning of `/users/list`.
+
+#### `state`
```

Comment:
Do you think this should be in supplemental rather than basic usage, given the restrictions using it, i.e. no support for IE11.

Reply:
I think this depends on how soon `state` will become the preferred way of interacting with the store.